### DISCUSSION TITLE
Can we detect when zoomin storyview has finished its animation?

### DISCUSSION THREAD

**Discussion by BurningTreeC** [TiddlyWiki/TiddlyWiki5#5928] [Q&A] [answered]
I have a problem with my muuri storyview plugin and the zoomin storyview.
I would need a way to detect when the zoomin storyview has finished its animation

Is there a way to do this?

**Comment by pmario** [comment]
We may probably add a new hook, to all of the story-views, th-story-animation-done or something similar

**Reply by BurningTreeC** [reply]
That would be great @pmario

**Comment by Jermolene** [comment, marked as answer]
Hi @BurningTreeC could you describe the problem in more detail? Such a hook seems like it would be very hard to use in situations where multiple storyviews are in use.

**Reply by BurningTreeC** [reply]
Ok @Jermolene , an example would be https://burningtreec.github.io/tiddlywiki-muuri when you set the storyview to `zoomin` and open the `Kanban` tiddler

The problem is, that the muuri storyview has finished its layout while the tiddler is still animating and that causes the muuri items to be positioned wrong

**Reply by Jermolene** [reply]
Hi @BurningTreeC I tried the following but couldn't see the problem:

1. Visited https://burningtreec.github.io/tiddlywiki-muuri/
2. Switched to the "Tools" tab and set the storyview to "zoomin"
3. Switched to the "Recent" tab and opened the "Kanban" tiddler

**Reply by BurningTreeC** [reply]
Thanks for testing @Jermolene - now I've seen that it only happens using firefox
